### ISSUE TITLE
Обновление russia-blacklist.txt

### ISSUE THREAD

**Issue by Iowerth** [Storik4pro/goodbyeDPI-UI#36]
Win 10 x64
Версия gDPI-UI 1.2.2
Движок - zapret v68
Пресет - 3. YTDisBystro

Какой бы из двух способов обновления russia-blacklist.txt ни выбрал - программа выводит уведомление о скачивании файла и бесконечно висит:
<details>
  <summary>Скриншот</summary>

![y5ENUMDkAx](https://github.com/user-attachments/assets/7678e80a-deda-4a29-ab68-5452080e78a8)
</details>

Однако, файлы скачиваются (в случае с p.thenewone.lol - 2.1 Мб, во втором случае - несколько Кб). Думаю, что это баг неотображения завершения скачивания файла.

**Comment by Storik4pro** [comment]
Да, это лишь визуальный баг. В зависимости от системы и нагрузки в данный момент уведомление может отобразится по разному
![image](https://github.com/user-attachments/assets/c8b9a98a-2d80-4d57-a03a-14f3d8104073)  > Правильный вариант.

Прямо сейчас с этим ничего нельзя сделать из-за определенных ограничений со стороны системы.
Я постепенно готовлюсь к переходу на C для повышения стабильности приложения и нормального доступа к системным функциям

**Comment by Iowerth** [comment]
Спасибо, так я и думал, ждем :)
